src/pudl/metadata/resources/glue.py

```diff
@@ -3,6 +3,23 @@
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {
     "epacamd_eia": {
+        "description": """This crosswalk table comes from the
+EPA's Github repo camd-eia-crosswalk:
+https://github.com/USEPA/camd-eia-crosswalk.
+It's purpose is to connect EPA units with EIA plants, boilers, and generators.
+The camd-eia-crosswalk README and our Data Source documentation page on
+:doc:`../data_sources/epacems` depict the complicated relationship between EIA and EPA
+data, specifically the nature of EPA vs. EIA "units" and the level of granularity that
+one can connect the two sources.
+
+The crosswalk table is generated using EIA data from 2018 meaning that any plants that
+have shifted before or since then aren't accurately reflected in the data. We're hoping
+to create a temporal version of the crosswalk at some point.
+
+Our version of the crosswalk clarifies some of the column names and removes unmatched
+rows. The :func:`pudl.glue.epacamd_eia.transform` function doc strings explain what
+changes are made from the EPA's version.
+""",
```

Comment:
In general, you can't use RST in any string and expect it to get rendered. Docstrings (which are parsed and rendered by the apidoc process in the docs build) and the dedicated RST files under the `docs/` directory (the main part of the docs build) will definitely get translated, but this string is just a Python string, which we **happen** to be using in the construction of our documentation.

This is only working because we're using this string to construct RST files dynamically under the `docs/` hierarchy, before the docs build happens, via some Jinja templates.

Was all the difficulty in getting the link to the EPA repo to work?  Was it your editor doing the reformatting, or did it happen in the pre-commit hooks?

Reply:
I figured I could write regular RST syntax as a string and would get translated in the `data_dictionary_metadata_to_rst` function, but it wasn't working when I tried creating a hyperlink for the EPA github: 

```
`EPA Github <https://github.com/USEPA/camd-eia-crosswalk>`__
```

Reply:
> Was it your editor doing the reformatting, or did it happen in the pre-commit hooks?

It would happen via a docs build failure.

Reply:
Somewhere in the Python string to Jinja to RST to HTML process the string was getting HTML encoded -- characters like < and > have special meanings in HTML and need to be escaped. My guess is it was in the Jinja templating, since Jinja is typically used to layout web pages.